pkgs/tools/virtualization/cloud-init/0001-add-nixos-support.patch

```diff
@@ -52,6 +73,8 @@ diff -ruN cloud-init-0.7.6.orig/cloudinit/distros/nixos.py cloud-init-0.7.6/clou
 +        # calls from repeatly happening (when they
 +        # should only happen say once per instance...)
 +        self._runner = helpers.Runners(paths)
++        self.usr_lib_exec = os.path.join(os.path.dirname(__file__),
++                                         "../../../../../libexec")
```

Comment:
where does that point to, the current-system libexec?

Reply:
`/usr/lib` if not overwritten. Not it points to the `$out/libexec`.